content_sync/apis/github.py

```diff
@@ -132,7 +137,9 @@ def upsert_content_file(
             return
         repo = self.get_repo()
         path = website_content.content_filepath
-        data = self.format_content_to_file(website_content)
+        data = ContentFileSerializerFactory.for_content(
+            site_config=self.site_config, website_content=website_content
+        ).serialize(website_content=website_content)
```

Comment:
I think that just adds a layer of obscurity. It's less code to read at the places where it's invoked, but I want to avoid having a helper method that just has a kitchen sink of arguments that are loosely related. I'm gonna tweak this a little b/c I agree it does seem a little more verbose than it needs to be